websites/C/Coub/presence.ts

```diff
@@ -312,26 +304,28 @@ const pages: PageContext[] = [
 	],
 	presence = new Presence({
 		clientId: "818598086984728576",
-	});
-
+	}),
+	IMAGES = {
```

Comment:
Shouldn't this be lowercase

Reply:
dunno, lint didnt show any warnings

Reply:
lint doesn't warn for that but that's not the capitalization standard we follow